### PR TITLE
Removed Quotes 

### PR DESCRIPTION
**Pull request by akshita2k**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -523,7 +524,23 @@ const data = [
...
+    },
+	{
+        longitude: 77.620540,
+	latitude: 13.079790,
+        batch: 2020,
+        photo: akshita2k,
+        region: "Asia",
+        skills: ["App Dev", "Web Dev", "AI/ML", "Cloud"],
+        name: "Akshita Sharma",
+        bio: "I enjoy learning and exploring new vistas of Technology while also sharing the knowledge among my peers!.",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:536]:
```suggestion
        bio: "I enjoy learning and exploring new vistas of Technology while also sharing the knowledge among my peers!",
```

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -523,7 +524,23 @@ const data = [
...
+        url: "https://dsc.community.dev/dayananda-sagar-academy-of-technology-and-management/",
+        chapterName: "Dayananda Sagar Academy of Technology and Management",
+        github: "https://github.com/akshita2k",
+        linkedin: "https://www.linkedin.com/in/akshita-sharma-15a703111",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:540]:
```suggestion
        linkedin: "https://linkedin.com/in/akshita-sharma-15a703111",
```

File: src/components/json/worldData.js [hunk 3]
```diff
@@ -523,7 +524,23 @@ const data = [
...
+        instagram: "https://www.instagram.com/akshi_4446",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:541]:
```suggestion
        instagram: "https://instagram.com/akshi_4446",
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me now. Thanks for adding yourself!

**Pull request merged**